ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -323,15 +325,28 @@ def _wan_load_balance_policies(self) -> list:
         control_plane_virtual_topology = get(self._hostvars, "wan_virtual_topologies.control_plane_virtual_topology", default={})
         name = get(control_plane_virtual_topology, "name", default="CONTROL-PLANE-PROFILE")
         wan_load_balance_policies = [self._generate_wan_load_balance_policy(f"{name}_LB", control_plane_virtual_topology, default_all=True)]
-        for avt_policy in get(self._hostvars, "wan_virtual_topologies.policies", []):
+        for avt_policy in self._filtered_wan_policies:
+            policy_name = avt_policy["name"] if avt_policy["name"] != self._default_vrf_policy["name"] else avt_policy["realname"]
```

Comment:
why not just have a _is_default boolean set in the _default_vrf_policy and then add the suffix in the few places needed?

Reply:
Point is you have more places where you jump hoops to find the correct name vs where  you actually benefit from the updated name.